client-src/elements/chromedash-drawer.js

```diff
@@ -261,6 +261,9 @@ export class ChromedashDrawer extends LitElement {
         ${this.renderNavItem('/metrics/css/animated', 'CSS Animation')}
         ${this.renderNavItem('/metrics/feature/popularity', 'JS/HTML')}
 
+        <div class="section-header">Reports</div>
```

Comment:
According to our current menu style, this should have an `<hr>` above it.  I plan to change the menu style but that hasn't happened yet.

FYI, this new "Reports" section seems like it would be interesting to a fairly narrow subset of users, so I think that when I redo the menu style using sl-tree, I will make this section initially collapsed.